docs/utilities/feature_flags.md

```diff
@@ -1,57 +1,366 @@
 ---
-title: Feature flags
-description: Utility
+title: Feature flags description: Utility
 ---
 
-The feature flags utility provides a simple rule engine to define when one or multiple features should be enabled depending on the input.
+The feature flags utility provides a simple rule engine to define when one or multiple features should be enabled
+depending on the input.
 
-!!! tip "For simpler use cases where a feature is simply on or off for all users, use [Parameters](parameters.md) utility instead."
+!!! tip "For simpler use cases where a feature is simply on or off for all users, use [Parameters](parameters.md)
+utility instead."
 
 ## Terminology
 
-Feature flags are used to modify a system behaviour without having to change their code. These flags can be static or dynamic.
+Feature flags are used to modify a system behaviour without having to change their code. These flags can be static or
```

Comment:
the static part is usually related to the part where they are not changed without redeploying the code. The fact that you have a rule engine is a different story. You can have dynamic toggles that dotn have rules. They are dynamic becuase they are changed/stored in another place untreated to the application.  
I think "global" would be the better term here

Reply:
The redeployment criteria is true for static flag in our case as well, just for the configuration instead of the code. If we have no rules the flag has no evaluation logic.